docs/developer-guide/contributing.md

```diff
@@ -1,31 +1,24 @@
 # Contributing code
 
-Before contributing code, please familiarise yourself with the [Project conventions](/developer-guide/project-conventions.md).
+Before contributing code, please familiarise yourself with the [Project conventions](project-conventions.md).
 
 ## General advice:
 
-- Submit pull-requests sooner rather than later: CI feedback is your friend, not your enemy
+- Submit pull-requests sooner rather than later: CI feedback is your friend, not your enemy.
 - Mark in-progress PRs as drafts until they are ready for review.
 - Don't be afraid to show your working. We're all learning. If you need help, linking to code changes in a PR is a quick and easy way to explain the problem.
 
 ## Submitting a pull request (PR)
 
 1. Push your branch to the remote.
 2. Head to https://github.com/nationalarchives/ds-wagtail/pulls and create a pull request from your branch.
-
-    For the PR Title:
-    - For ticketed features or ticketed bugs, use the naming convention: `DF-XXX: Ticket name / short description`.
-    - For housekeeping tasks or other unticketed work, use the convention: `Chore: Short description`.
-3. To mark a PR as a draft, look for the **"Convert to draft"** link (after submitting) and click on it.
+    1. For the PR Title:
```

Comment:
We can avoid this bullet number 1 with Shift+Enter and tabs. Then re-number others.